schema.graphql

```diff
@@ -10,20 +10,23 @@ type Domain @entity {
   isLocked: Boolean!
   metadata: String
   royaltyAmount: BigInt!
+  timeStamp: BigInt
```

Comment:
`timeStamp` is pretty ambiguous, sure it's a timestamp, but what is it a timestamp of? I'm assuming the creation, but we shouldn't leave this up for assumption and be very clear about it. So can you please:

- Rename to `creationTime` or `creationTimestamp` or `createdAt`

Another note: 'timestamp' is one word, so you don't need to capitalize the 's'